samples/microsoft/infrastructure-setup/15-private-network-standard-agent-setup/deleteCaphost.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+# Script to delete the capability host
+
+# Prompt for required information
+read -p "Enter Subscription ID: " subscription_id
+read -p "Enter Resource Group name: " resource_group
+read -p "Enter Hub or Project name: " workspace_name
+read -p "Enter CapabilityHost name: " caphost_name
+
+# Get Azure access token
+echo "Getting Azure access token..."
+access_token=$(az account get-access-token --query accessToken -o tsv)
+
+if [ -z "$access_token" ]; then
+    echo "Error: Failed to get access token. Please make sure you're logged in with 'az login'"
+    exit 1
+fi
+
+# Construct the API URL
+api_url="https://management.azure.com/subscriptions/${subscription_id}/resourceGroups/${resource_group}/providers/Microsoft.CognitiveServices/accounts/${account_name}/capabilityHosts/${caphost_name}?api-version=2025-10-01-preview"
+
+# Check if the curl command was successful
+if [ $? -ne 0 ]; then
```

Comment:
nit: you may (or may not) want to use the safeguard `set -euo pipefail` to make the script stop automatically if any of the commands fail or any variables are missing.

The drawback is that it gets harder to write custom error handling for the particular errors.